### PR TITLE
Memurai Developer v2.0.5

### PR DESCRIPTION
**Pull request by MemuraiUser**
Pull request was created by azure pipeline

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37095)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This manifest is not v1.0.0 or v1.1.0 compliant and is written weirdly.

Please use YAMLCreate.ps1 or wingetcreate (which can be used in CI/CD to automate package updates) to create your package.

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+PackageIdentifier: Memurai.MemuraiDeveloper
+PackageVersion: 2.0.5
+PackageName: MemuraiDeveloper
```
Comment (ItzLevvie) [inline manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:3]:
```suggestion
PackageName: Memurai Developer
```

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Publisher: Memurai
```
Comment (ItzLevvie) [inline manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:4]:
```suggestion
Publisher: Janea Systems
```

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- cache
+- redis-compatible
+- redis-client
+InstallModes:
+ - silent
+Installers:
+  - Architecture: x64
+    InstallerType: msi
+    InstallerUrl: https://dist.memurai.com/releases/Memurai-Developer/2.0.5/Memurai-Developer-v2.0.5.msi
+    InstallerSha256: d88ee77a2fc5063e979639084b1797bcce7cbfc40fee2617866f4ebda74e280b
```
Comment (ItzLevvie) [inline manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:29]:
```suggestion
    InstallerSha256: d88ee77a2fc5063e979639084b1797bcce7cbfc40fee2617866f4ebda74e280b
    ProductCode: "{70E00C1E-673F-4FCB-A20C-ECBD5A8FE747}"
```

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:4]: This must match Control Panel.

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:3]: This must match Control Panel.

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:29]: This must match Windows Registry.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml (hunk 3 above)
Comment (MemuraiUser) [reply manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml:29]: Added ProductCode as suggested.

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml (hunk 2 above)
Comment (MemuraiUser) [reply manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml]: Done

File: manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml (hunk 1 above)
Comment (MemuraiUser) [reply manifests/m/Memurai/MemuraiDeveloper/2.0.5/Memurai.MemuraiDeveloper.yaml]: Done

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**